desktop-widgets/printdialog.cpp

```diff
@@ -110,16 +117,21 @@ PrintDialog::PrintDialog(bool inPlanner, QWidget *parent) :
 	QPushButton *printButton = new QPushButton(tr("P&rint"));
 	connect(printButton, SIGNAL(clicked(bool)), this, SLOT(printClicked()));
 
+	// To do: Port preview to WebEngine. For the time being: Use OS print preview
+#ifndef USE_WEBENGINE
 	QPushButton *previewButton = new QPushButton(tr("&Preview"));
 	connect(previewButton, SIGNAL(clicked(bool)), this, SLOT(previewClicked()));
+#endif
```

Comment:
admittedly there is the question of "shouldn't print preview be an OS function in the first place?"
I find it really weird that every app should have to implement this, instead of having it in the OS print dialog. This seems to work very well on macOS, but simply isn't there on many Linux flavors (not sure - maybe it works on some but standard KDE/Plasma doesn't appear to have a print preview as part of the print dialog).
So yeah, unfortunately we might need this.